CMakeLists.txt

```diff
@@ -87,7 +87,6 @@ file(GLOB_RECURSE SOURCE_FILES
 )
 list(APPEND SOURCE_FILES
     ${ark_SOURCE_DIR}/src/Ark.cpp
-    ${ark_SOURCE_DIR}/src/Log.cpp
```

Comment:
because it was useless: it was designed to handle console output and file output, but the latter was never used and would never have been used. Also taking so much space and template functions just for prefixing log output and adding colors wasn't worth it

Reply:
I see